bookkeeper-server/src/main/java/org/apache/bookkeeper/client/BookKeeper.java

```diff
@@ -1058,9 +1056,7 @@ public LedgerHandle createLedgerAdv(int ensSize, int writeQuorumSize, int ackQuo
     public void asyncCreateLedgerAdv(final int ensSize, final int writeQuorumSize, final int ackQuorumSize,
             final DigestType digestType, final byte[] passwd, final CreateCallback cb, final Object ctx,
             final Map<String, byte[]> customMetadata) {
-        if (writeQuorumSize < ackQuorumSize) {
-            throw new IllegalArgumentException("Write quorum must be larger than ack quorum");
-        }
+        checkLedgerCreationParameters(ensSize, writeQuorumSize, ackQuorumSize);
```

Comment:
Shall move the check logic to `LedgerCreateOp#initiate`; that's the only one entrace.

Reply:
+1

Reply:
Make sense.